runusb

```diff
@@ -256,6 +264,11 @@ def main(args=sys.argv[1:]):
     def close_process(path, process):
         # With the mountpoint now missing the only sensible thing to do is
         # to die horribly.
+        print(get_machinename(path))
+        command = ('machinectl', 'terminate', get_machinename(path))
```

Comment:
If the `print` is intended to remain (and isn't a debug artefact), I'd prefer use of a variable to ensure these can't get out of sync. Else remove the `print`.